src/devices/Vcnl4040/samples/Simple/.gitignore

```diff
@@ -0,0 +1 @@
+output
```

Comment:
Same here: Sample output should be below the `bin` folder, so that no extra ignore is necessary.

Reply:
This is an artifact of my own deployment script. I'll remove it.